server/swap/swap_test.go

```diff
@@ -1256,8 +1410,12 @@ func TestTxWaiters(t *testing.T) {
 
 	// Get the MatchNotifications that the swapper sent to the clients and check
 	// the match notification length, content, IDs, etc.
-	ensureNilErr(rig.ackMatch_maker(true))
-	ensureNilErr(rig.ackMatch_taker(true))
+	if err := rig.ackMatch_maker(true); err != nil {
+		t.Fatal(err)
+	}
+	if err := rig.ackMatch_taker(true); err != nil {
+		t.Fatal(err)
+	}
```

Comment:
I got frustrated by `ensureNilErr` among other helpers because the error/fatal line number was in side the helper function rather than the line where the helper was called, making debugging the test quite difficult.

Reply:
I recently found the method off of testing `t.Helper()` which may allow helpers to be used while not messing up the line numbers.
https://golang.org/pkg/testing/#B.Helper

Reply:
Wow, nice find @JoeGruffins